src/runtime_src/core/include/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(XRT_HEADER_SRC
   xrt.h
   xcl_app_debug.h
   xcl_macros.h
+  xcl_graph.h
```

Comment:
No, this header should not be exported.  We don't want end applications to use xcl APIs.

Reply:
Removed it from the include list